### ISSUE TITLE
[BUG] plugin fails to load: undefined symbol

### ISSUE THREAD

**Issue by schradert** [outfoxxed/hy3#155]
Currently on `nixos-unstable` with `hyprland` and `hy3` flake inputs pinned to `v0.45.0` and `hl0.45.0` refs respectively. I get an alert that the plugin fails to load, and running `hyprctl plugin load /nix/store/jl0ai2q4sp5l7nyvs33pn7djb40bnvrv-hy3-hl0.45.0/lib/libhy3.so` gives `undefined symbol: _ZN11CCompositor30getFullscreenWindowOnWorkspaceERKl`.

**Comment by schradert** [comment]
When I pin each back to `0.44.0`, it's a different undefined symbol: `_ZN13CHyprRenderer13damageMonitorEP8CMonitor`... I feel like I must be missing something, but I don't know what.

**Comment by ZWORX52** [comment]
I just made a PR #156 that changes a few function calls, as the existing code called functions that were removed from Hyprland in a git commit. (I noticed this because hy3 wouldn't build). `_ZN11CCompositor30getFullscreenWindowOnWorkspaceERKl` is the mangled name for `CCompositor::getFullscreenWindowOnWorkspace(long const&)`, which was the name of one of the methods I changed from (that no longer existed).

In non-programming terms, this is related to #156 and could be fixed by it.
